config.json

```diff
@@ -16,6 +16,7 @@
 
   ],
   "exercises": [
+    { "difficulty" : 1, "slug" : "allergies", "topics" : [] },
```

Comment:
Could we move this somewhere else in the array of exercises? If I'm not mistaken, the order here affects how they're listed on the site, and as the Bob exercise is mentioned in examples it's probably best for us to keep that one at the top. Also, could we keep the formatting consistent with the other objects in this array?